earth_enterprise/src/server/gestream_tables.sql

```diff
@@ -115,13 +115,19 @@ BEGIN
     alter table target_db_table add primary key (target_id, db_id);
   END IF;
 
-  IF NOT check_table_exists('publish_context_table') THEN
+
+  IF check_table_exists('publish_context_table') THEN
```

Comment:
Oh boy, this also affects restoring a database backup since we have to also support backups from older versions.